build.ps1

```diff
@@ -22,10 +22,6 @@ Specifies the amount of information to be displayed.
 Shows description about tasks.
 .PARAMETER DryRun
 Performs a dry run.
-.PARAMETER Experimental
```

Comment:
All these changes are from bringing over the new version of the file from  https://github.com/cake-build/resources with the exception of ONE line at the end where I am dumping (almost) all the environment variables